ogr/services/base.py

```diff
@@ -20,13 +20,16 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from typing import List, Optional, Match, Any
+from typing import List, Optional, Match, Any, TypeVar
 
-from ogr.abstract import GitService, GitProject, PRComment, GitUser
+from ogr.abstract import GitService, GitProject, PRComment, GitUser, IssueComment
 from ogr.parsing import parse_git_repo
 from ogr.utils import search_in_comments, filter_comments
 
 
+Comment = TypeVar("Comment", IssueComment, PRComment)
```

Comment:
I would define a common "superclass" for `IssueComment` and `PRComment`. What do you think?

Reply:
It can be handy in the future.

Reply:
I wanted to do that in #230, so I used this just as quick fix for now.